helpers/focus.js

```diff
@@ -62,6 +63,22 @@ export function getFocusableDescendants(node, options) {
 export function getFocusPseudoClass() {
 	return isFocusVisibleSupported() ? 'focus-visible' : 'focus';
 }
+export function getFocusRingStyles(selector, { applyOnHover = false, noFocusPseudoClass = false, baseOffset = '2px', extraStyles = '' } = {}) {
+	const baseSelector = selector;
+	if (!noFocusPseudoClass) selector += `:${getFocusPseudoClass()}`;
+	if (applyOnHover) selector += `, ${baseSelector}:hover`;
+	const cssSelector = unsafeCSS(selector);
+	return css`${cssSelector} {
+		${unsafeCSS(extraStyles)}
```

Comment:
The `extraStyles` feels a little wrong in that we're losing CSS linting, etc. since they're just strings. Would this still work if they were passed in as CSS literals?